docs/docs/noir/standard_library/meta/quoted.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Quoted
+---
+
+`std::meta::quoted` contains methods on the built-in `Quoted` type which represents
+quoted token streams and is the result of the `quote { ... }` expression.
+
+## Methods
+
+### as_expr
+
+#include_code as_expr noir_stdlib/src/meta/quoted.nr rust
+
+Parses the quoted token stream as an expression. Returns `None` if
+the expression failed to parse.
+
+Example:
+
+```rust
+comptime {
```

Comment:
Do we have code examples from our test programs we can include here? Same with the other examples? Non-blocking though

Reply:
Done